src/main/java/net/mcreator/blockly/java/blocks/TimeToStringBlock.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2021, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.blockly.java.blocks;
+
+import net.mcreator.blockly.BlocklyToCode;
+import net.mcreator.blockly.IBlockGenerator;
+import net.mcreator.generator.template.TemplateGeneratorException;
+import org.w3c.dom.Element;
+
+public class TimeToStringBlock implements IBlockGenerator {
+
+	@Override public void generateBlock(BlocklyToCode master, Element block) throws TemplateGeneratorException {
+		master.append("Calendar.getInstance().getTime().toString()");

```

Comment:
Maybe consider adding another block like this one "to formatted string" that takes string format input eg. "YYYY MM DD"

Reply:
How do I convert a String to a Date? There is no format method in `Date` or `Calendar`.

Reply:
> Also, I don't see the interest to have a block converting a date gave by the user to a formatted String

In my career, in 95% of cases I wanted custom date format.

Reply:
`new SimpleDateFormat("yyyy-MM-dd").parse(Calendar.getInstance().getTime())` ;)